src/EThree.ts

```diff
@@ -167,21 +176,26 @@ export default class EThree {
             argument.push(ownPublicKey);
         }
 
-        let res: Data = this.virgilCrypto.signThenEncrypt(message, privateKey, argument);
-        if (isString) res = res.toString('base64');
+        const res: Data = this.virgilCrypto.signThenEncrypt(message, privateKey, argument);
+        if (isMessageString) return res.toString('base64');
+        if (isArrayBuffer) return toArrayBuffer(res as Buffer);
```

Comment:
This is a breaking change. Are you sure it is necessary?

Reply:
Yeah, why we should use buffer if user work with ArrayBuffer, the main goal of that you give some format you receive that format. And that it return Buffer it is bug, not breaking change.

Reply:
Why is that a bug if you explicitly tell that the method returns a Buffer? You return a new value, why should it be of the same type as the input? 
Anyway, my point is - the following code will stop working as intended after upgrade:
```
const encrypted = eThree.encrypt(new ArrayBuffer(), publicKeys);
// will log `"[object ArrayBuffer]"` instead of ciphertext in base64
console.log(encrypted.toString('base64'));
```
Which is how we typically suggest our users to serialize the ciphertext to transmit it to the recipient over the network.

Reply:
And why is this _API_ change a part of `migrate from jest to karma+mocha` pull request anyway?

Reply:
Because if you receive the same type you can do the same operations as you did before encryption. So it makes integration easier you just encrypt and do the same operation with returned value.
And the migration to karma show me that bug, so I fix it already. Why do i need commit pull request with test which are shell not pass?

Reply:
And package.json version is 0.x.x it means that it can contain breaking change, I just need add it to changelog

Reply:
What test? Why did you expect it to return an ArrayBuffer? This is not a bug. The method's signature clearly says that it returns a Buffer. If that doesn't match your expectations, maybe your expectations are wrong.

Anyway, what if someone called `toString('base64')` on the returned value of `encrypt` when passing it an ArrayBuffer? Do we just tell them it was a bug that it worked properly?

Reply:
If you "can" make braking changes, it doesn't mean you should.

Reply:
We tell them it should work in that way. What if user already working with ArrayBuffer, he already using some operations with instances of this object and now he need include Buffer package, change application logic just to handle encrypt output

Reply:
He doesn't need to include the Buffer package. And `Buffer` instances have `.buffer` property of type `ArrayBuffer`

Reply:
Okay, now that makes more sense for me, thank you for explanation! I will revert this back to return Buffer.